src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,12 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService temp = new UserService();
+        if (temp.findByEmail(email) != null) {
+            if (temp.findByEmail(email).getPassword().equals(password)) {
+                return true;
+            }
+        }
         return false;
```

Comment:
we can write this code in one line. Also, let's make a local variable for the result of the findByEmail method so as not to call it twice - this can affect the performance of the program

Reply:
done 🤝🤝